### PR TITLE
update smart contract addresses for goerli testnet

### PR DESCRIPTION
**Pull request by tarieuler**

### CODE REVIEW

File: addresses/euler-addresses-goerli.json [hunk 1]
```diff
@@ -1,24 +1,83 @@
...
-    "euler": "0x95E54F5B6bCbFE101EC178c701E98de8FF1ff4a4",
-    "installer": "0x35ED43792532C19c44bE0A8fF9738bAAcF0991CC",
-    "markets": "0x876b9CF10471c12EF5Eb90BC0A32a6EBa115dB80",
-    "liquidation": "0x10feBCfF158139d532c2240231D7d47F6c3860d2",
-    "governance": "0x869fB401Af7f9E166b32E5cA4E87F3Bc8EA97fA8",
-    "exec": "0xFEC37aD6BC17E825fff4631230ae039711bDfEf0"
+    "uniswapV3Factory": "0xdA4087686D519b090C2F3Fd6d9Fa6Af3D3990124",
+    "swapRouter": "0x7862037d641112804801b819DE980C51fB54940F",
+    "invariantChecker": "0x6C5AA3Bd6Dc159a6730FCF7FF3b7a2602AEE620b",
+    "flashLoanNativeTest": "0x2D934E27e62f5084293CB0cA146F8Cd216eF1696",
```
Comment (kasperpawlowski) [inline addresses/euler-addresses-goerli.json:68]: do we need those test addresses?

File: hardhat.config.js [hunk 2]
```diff
@@ -86,4 +86,10 @@ for (let k in process.env) {
             }
         }
     }
+
+    if (k == "ETHERSCAN_API_KEY") {
```
Comment (hoytech) [inline hardhat.config.js:90]: Not needed here

Comment (tarieuler) [reply hardhat.config.js]: Thanks. Resolved

File: addresses/euler-addresses-goerli.json (hunk 1 above)
Comment (tarieuler) [reply addresses/euler-addresses-goerli.json]: Thanks. Resolved

**Pull request merged**